cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
There are many terrifying aspects of how we make and deploy and use containers. At some point we should sit down and rationalize it and put in procedures and checks etc.